next.config.jsのエラーを修正

next.config.jsで以下のエラーが出る
```
Parsing error: Cannot find module 'next/babel'
Require stack:
- .../lottery/view/node_modules/next/dist/compiled/babel/bundle.js
- .../lottery/view/node_modules/next/dist/compiled/babel/eslint-parser.js
- .../lottery/view/node_modules/eslint-config-next/parser.js
- .../lottery/view/node_modules/@eslint/eslintrc/dist/eslintrc.cjseslint
```

--- a/view/.eslintrc.json
+++ b/view/.eslintrc.json
@@ -1,6 +1,10 @@
 {
   "extends": [
     "next/core-web-vitals",
-    "plugin:storybook/recommended"
+    "plugin:storybook/recommended",
+    "next", 
+    "next/core-web-vitals", 
+    "prettier", 
+    "next/babel"
   ]
 }

--- a/view/next.config.mjs
+++ b/view/next.config.mjs
@@ -7,4 +7,4 @@ const nextConfig = {
   }
 }
 
-module.exports = nextConfig
+export default nextConfig

